app/2.0/start/toolbox/deploy.md

```diff
@@ -143,10 +162,33 @@ guide](https://www.firebase.com/docs/hosting/quickstart.html).
     ]
     ```
 
+    For example, your `firebase.json` file may look like this afterwards:
+	
+    ```
+    {
+      "database": {
+        "rules": "database.rules.json"
+      },
+      "hosting": {
+        "public": "build/default",
+        "rewrites": [
+          {
+            "source": "!/__/**",
+            "destination": "/index.html"
+          },
+          {
+            "source": "**/!(*.js|*.html|*.css|*.json|*.svg|*.png|*.jpg|*.jpeg)",
```

Comment:
I'm not sure about an alternative, but it makes more sense to me to use something like this (not sure if this is valid though):
```js
"source": "**/!*.(html|css|js|json|svg|png|jpg|jpeg)",
```